codebuild/common-posix.sh

```diff
@@ -3,42 +3,45 @@
 set -e
 
 CMAKE_ARGS="$@"
+BUILD_PATH="/tmp/builds"
+mkdir -p $BUILD_PATH
+INSTALL_PATH="$BUILD_PATH/install"
+mkdir -p $INSTALL_PATH
 
 function install_library {
+    pushd
+    cd $BUILD_PATH
     git clone https://github.com/awslabs/$1.git
     cd $1
 
     if [ -n "$2" ]; then
         git checkout $2
     fi
 
-    mkdir build
-    cd build
-
-    cmake -DCMAKE_INSTALL_PREFIX=../../install -DENABLE_SANITIZERS=ON $CMAKE_ARGS ../
+    cmake -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH -DCMAKE_PREFIX_PATH=$INSTALL_PATH  -DENABLE_SANITIZERS=ON $CMAKE_ARGS ./
```

Comment:
It may make stuff cleaner if you can put `-DCMAKE_INSTALL_PREFIX=$INSTALL_PATH -DCMAKE_PREFIX_PATH=$INSTALL_PATH  -DENABLE_SANITIZERS=ON` into `CMAKE_ARGS`